ci/docker/install/ubuntu_scala.sh

```diff
@@ -29,12 +29,6 @@ apt-get update
 apt-get install -y openjdk-8-jdk
 apt-get install -y openjdk-8-jre
 
-echo "deb https://dl.bintray.com/sbt/debian /" | tee -a /etc/apt/sources.list.d/sbt.list
```

Comment:
Could you elaborate?

Reply:
The reason to remove sbt and Scala is we don't need these components to run a maven project. There is Scala compiler embedded in the maven project. In that case, sbt and Scala installation seemed to be unnecessary